common/GNUmakefile

```diff
@@ -192,6 +204,15 @@ ifeq ($(TARGET),uefi-riscv64)
         -z text
 endif
 
+ifeq ($(TARGET),uefi-loongarch64)
+    override LDFLAGS_FOR_TARGET += \
+        -m elf64loongarch \
+        -static \
+        -pie \
+        --no-dynamic-linker \
```

Comment:
Remove `--no-dynamic-linker` and `-static` please, since those are no longer needed, similar to other ports.